src/main/java/ratpack/pac4j/RatpackPac4j.java

```diff
@@ -41,28 +47,23 @@
 import ratpack.pac4j.internal.RatpackWebContext;
 import ratpack.path.PathBinding;
 import ratpack.registry.Registry;
-import ratpack.session.Session;
 import ratpack.util.Types;
 
-import java.util.Arrays;
-import java.util.List;
-import java.util.Optional;
-
-import static java.util.Arrays.asList;
-
 /**
  * Provides integration with the <a href="http://www.pac4j.org">Pac4j library</a> for authentication and authorization.
  * <p>
- * Pac4j support many different authentication providers, such as external sources like GitHub, Twitter, Facebook etc., as well
- * as proprietary local authentication sources.
+ * Pac4j support many different authentication providers, such as external sources like GitHub, Twitter, Facebook etc.,
+ * as well as proprietary local authentication sources.
```

Comment:
Various formatting only changes to undo.  Javadoc line wrapping, ordering of imports, etc.

Reply:
I'll take another look after you've cleaned this up to only include the changes you actually care about.